check_signed_off.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -x
+
+echo "Checking range: ${TRAVIS_COMMIT_RANGE}"
+
+for i in $( (git rev-list --no-merges "$TRAVIS_COMMIT_RANGE") )
+do
+    COMMIT_MSG="$(git show -s --format=%B "$i")"
+    COMMIT_USER_EMAIL="$(git show -s --format="%an <%ae>" "$i")"
+    echo "$COMMIT_MSG" | grep "^Signed-off-by: ${COMMIT_USER_EMAIL}" >/dev/null
```

Comment:
Haha, np. 
I changed the line by replace grep to grep -F, and and dumped the '^'